components/spinner/test/index.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+
+/**
+ * Internal dependencies
+ */
+import Spinner from '../';
+
+describe( 'Spinner', () => {
+	describe( 'basic rendering', () => {
+		it( 'should match the following element', () => {
+			const element = <span className="spinner is-active" />;
+			const spinner = shallow( <Spinner /> );
+			expect( spinner.matchesElement( element ) ).to.be.true();
```

Comment:
These kind of tests are also useful for when you need to do upgrades, so you can know a lot of what is broken from the unit tests themselves.